### ISSUE TITLE
Improvements in Guide Requirements and Use

### ISSUE THREAD

**Issue by kiraitachi** [secops4thewin/TA-intezer#1]
Hello. 

I'm a security reasearcher with long experiencie in use with different technologies such as Virustotal, Hybrid-Analys and JoeSanbox, etc and implementing them in Splunk.

Although when I tried this addon, I find it difficult to make it work. The info shown in the Readme.md is no linked in Splunkbase and no Github URL is remarked in Splunkbase, dunno if that has anything to do with you, but it would help other newcomers like me.

Besides that, the info in the Readme.md falls short, there is no example of use or screenshots...I would love to use it, but have no How-to on to do so, maybe thats why only 2 installations for this addon...

![image](https://user-images.githubusercontent.com/40744673/65139773-7a222980-da0d-11e9-9d31-7dd021a68176.png)

In short words...could you update the Read.md with some Guidelines examples on how to use it for newcomers like me?

Thanks in advance!
